This patch is needed, otherwise the sanity test for cpp fails on cross builds:

configure:5267: aarch64-linux-gnu-cpp    -Wall -Wshadow -g -pedantic-errors -I/usr/include -I/usr/local/include conftest.c
In file included from /usr/include/features.h:489,
                 from /usr/include/bits/libc-header-start.h:33,
                 from /usr/include/limits.h:26,
                 from conftest.c:13:
/usr/include/gnu/stubs.h:7:11: fatal error: gnu/stubs-32.h: No such file or directory
    7 | # include <gnu/stubs-32.h>
      |           ^~~~~~~~~~~~~~~~
compilation terminated.

Normally, this isn't an issue with the compiler itself, because we have a
wrapper that discards bad -I flags, but we don't have a cpp wrapper.

diff --git a/configure.ac b/configure.ac
index 40569655..5a64324a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -24,7 +24,6 @@ AC_SUBST(QUICKCHECKDIR)
 AX_CXX_COMPILE_STDCXX_17([noext], [mandatory])
 
 CPPFLAGS="$CPPFLAGS -Wall -Wshadow -g -pedantic-errors"
-CPPFLAGS="$CPPFLAGS -I/usr/include -I/usr/local/include"
 
 AC_CHECK_SIZEOF([void *])
 CXXFLAGS="$CXXFLAGS -DSIZEOF_VOID_P=$ac_cv_sizeof_void_p"
